modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
++
+[NOTE]
+====
+When creating the child compartment, specify the default parent compartment or any other parent compartment from the list.
+==== 
+
+. Create a bucket resource for your child compartment. Ensure that you specify your child compartment in the **Create in compartment** field for the bucket resource. Go to **Object Storage & Archive Storage > Buckets** and create a bucket, where **Bucket name** refers to your cluster's name.
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[**Install OpenShift with the Assisted Installer**] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required Assisted Installer steps. Ensure that for the **Cluster Details** step, you complete the following fields:
++
+.. **Cluster name**: Specify the name of your cluster, such as `ocidemo`. 
+.. **Base domain**: Specify the base domain of the cluster, such as `splat-oci.devcluster.openshift.com`. Provided you previously created a compartment on {oci}, you can get this information by going to **DNS management > Zones > List scope** and then select the parent compartment. Your base domain should show under the **Public zones** tab.
+.. **OpenShift version**: Specify OpenShift 4.15 or a later version.
+.. **CPU architecture**: Specify `x86_64` or `Arm64`.
+.. **Integrate with external partner platforms**, where you must specify `Oracle Cloud Infrastructure`. After you specify this value, the *Include custom manifests* checkbox is automatically selected. 
++
+[IMPORTANT] 
+====
+You can keep the default settings for the **Operators** step.
+====
+
+. For the **Host Discovery** step, click on the **Add hosts** button to display a dialog box. For the **SSH public key** field, add your SSH key from your local system. Click the **Generate Discovery ISO** button to generate the discovery image ISO file. Ensure that you download the file to your local system.
++
+[TIP]
+====
+You can create an SSH authentication key pair by using the `ssh-keygen` tool. 
+====
+
+. On {oci}, go to **Resources > Objects** and then click the **Upload** button. Upload the Agent ISO image to the bucket, and then click the **Upload** button to complete the upload operation.
+
+. From the **Objects** menu, locate your uploaded image, and then click the stacked dots icon. From the **Create Pre-Authenticated Request** window, select the **Object** tile. After you create the request, copy the URL from the **Pre-Authenticated Request URL** field. See link:https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/usingpreauthenticatedrequests_topic-To_create_a_preauthenticated_request_for_all_objects_in_a_bucket.htm#:~:text=Open%20the%20navigation%20menu%20and,Click%20Create%20Pre%2DAuthenticated%20Request.[Creating a Pre-Authenticated Request] in the Oracle documentation. 
```

Comment:
Please review the prepositions and verbs for UI elements.

Reply:
If you have another term in mind, please share. "Menu item" might be the only workable solution.

![Screenshot from 2024-01-10 16-35-34](https://github.com/openshift/openshift-docs/assets/57954076/e36981c2-096d-4367-bbe7-1a4092fc9ed0)